flyteidl/protos/flyteidl/service/agent.proto

```diff
@@ -12,4 +12,6 @@ service AsyncAgentService {
   rpc GetTask (flyteidl.admin.GetTaskRequest) returns (flyteidl.admin.GetTaskResponse){};
   // Delete the task resource.
   rpc DeleteTask (flyteidl.admin.DeleteTaskRequest) returns (flyteidl.admin.DeleteTaskResponse){};
+  // Do a job and get response from the agent server.
+  rpc DoTask (flyteidl.admin.DoTaskRequest) returns (flyteidl.admin.DoTaskResponse){};
```

Comment:
Sorry I did not pay much attention to this change. This service is named as `AsyncAgentService` but this RPC is for sync execution. Do you think it makes sense to have another service or renaming `AsyncAgentService` to `AgentService` (we will again need to think about breaking change if we rename it)?